relay-server/src/actors/spooler/mod.rs

```diff
@@ -1,3 +1,30 @@
+//! This module contains the [`BufferService`], which responsible for spooling of the incoming
+//! envelopes, either in memory or on persistent storage.
+//!
+//! The main entry point for the [`BufferService`] is the [`Buffer`] interface. Which currently
+//! supports:
```

Comment:
```suggestion
//! The main entry point for the [`BufferService`] is the [`Buffer`] interface, which currently
//! supports:
```
(not sure about this one)